src/components/SearchResults.astro

```diff
@@ -0,0 +1,146 @@
+---
+import payloadFetch from "@/utilities/payload-fetch";
```

Comment:
The `Layout` is already querying the site config data so let's refactor the two queries into `src/utilities/queries.ts` and also add the `?draft=true` only when `PREVIEW_MODE` is true.